pyproject.toml

```diff
@@ -0,0 +1,99 @@
+[project]
+name = "snuba"
+version = "25.8.0.dev0"
```

Comment:
this was previously in setup.py which i removed (we aren't building sdists or wheels?) - what was using the version from setup.py, craft? not sure yet